partcad/src/partcad/project.py

```
@@ -1276,23 +1276,32 @@
-            obj = self.get_sketch(name)
-            tasks.extend(asyncio.create_task(getattr(t, test_method)(tests, ctx, obj)) for t in tests)
-        for name in self.part_configs.keys():
-            obj = self.get_part(name)
-            tasks.extend(asyncio.create_task(getattr(t, test_method)(tests, ctx, obj)) for t in tests)
-        for name in self.assembly_configs.keys():
-            obj = self.get_assembly(name)
-            tasks.extend(asyncio.create_task(getattr(t, test_method)(tests, ctx, obj)) for t in tests)
-        results = await asyncio.gather(*tasks)
-        return False not in results
+        def get_objects(config_dict, getter):
+            for name in config_dict.keys():
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)